doc/scripts/docgen.py

```diff
@@ -54,6 +54,10 @@ def mkdir(path):
     pythonpath = os.pathsep.join([throot, pythonpath])
     sys.path[0:0] = [throot]  # We must not use os.environ.
 
+    # Make sure we don't use gpu to compile documentation
+    env_th_flags = os.environ['THEANO_FLAGS']
```

Comment:
This seems to cause an error if the env variable THEANO_FLAGS hasn't been initialized.